docs/src/apps/rent.md

```diff
@@ -1 +1,45 @@
-# Accounts and Rent
+# Storage Rent for Accounts
+
+Keeping accounts alive incurs storage cost called _rent_ because the cluster must actively maintain the data to process any future transactions on it. This is not like Bitcoin or Ethereum, where account storages don't incur any costs.
+
+The rent is debited from account's balance by the runtime upon the first access in the current epoch by transactions or once per an epoch if there is no transaction. The rent is up-front for the next epoch. If an account can't pay rent for the next epoch, it's purged immediately. The fee is currently a fixed rate, measured in bytes-times-epochs. The fee may change in the future.
```

Comment:
"...if there is no transaction" -> "...if there are no transactions"
"...rent is up-front for the next epoch..." -> "..rent is paid up front for the next epoch..."
To be clear, is rent checked at the beginning of each epoch?  So if an account can't pay rent it will be purged immediately at the start of a new epoch?

Reply:
Thanks for correction!

> So if an account can't pay rent it will be purged immediately at the start of a new epoch?

_In principle_, yes. Sadly, current implementation has some edge cases: https://github.com/solana-labs/solana/issues/10426